### ISSUE TITLE
rpm/deb packages

### ISSUE THREAD

**Issue by varuzam** [postfixadmin/postfixadmin#250]
Hello
I found packages here https://sourceforge.net/projects/postfixadmin/files/postfixadmin. but that repo looks abandoned
is there a plan to provide rpm/deb packages for new releases?

**Comment by DavidGoodwin** [comment]
Hi - Historically I've created .deb packages for each release, and then @cboltz creates rpms somehow and uploads them to sourceforge.

When I created the 3.2.1 and 3.2.2 releases, I didn't bother going through that process (perhaps I should have) as the Debian people picked 3.2.1 up quite quickly ( https://packages.debian.org/search?keywords=postfixadmin )

**Comment by neuffer** [comment]
Debian picks it up whenever I poke a friend of mine to update the Debian package. I did that the last time just before the frozen zone for the current Debian release began. He is reluctant to NMU it, if there is not an official release with a version number change. That is why I've advocated in the past for doing releases and not just pushing update after update into the SVN, now GIT repository.

The original maintainer has pretty much abandoned the package for the last I don't know how many years, but unfortunately still has not orphaned it.
@DavidGoodwin if we have a good set of changes and there is a release, I'll poke him again to push the update into Debian SID.
